packages/discovery-provider/src/tasks/index_payment_router.py

```diff
@@ -92,6 +93,8 @@
 # The amount of USDC that represents one USD cent
 USDC_PER_USD_CENT = 10000
 
+RECOVERY_MEMO_STRING = "recovery"
```

Comment:
maybe we could just use RouteTransactionMemoType instead of a separate var?

Reply:
Feel like slightly different purposes to me. This is used as a constant check against decoded data to determine which enum value to use, and leaves us free to change that enum value if needed. The other case (purchase) doesn't and couldn't compare decoded data against an enum value because it's based on structure of the resulting string. I think I'd rather not mix things so that some cases are checking directly against an enum and others are not, if that makes sense?